lib/msal-browser/src/config/Configuration.ts

```diff
@@ -88,6 +88,7 @@ export type BrowserSystemOptions = SystemOptions & {
 };
 
 export type ExperimentalOptions = {
+    enableExperimentalApi?: boolean;
```

Comment:
change to enable

Reply:
possible hang off of `system` instead of `experimental`?

Reply:
chose not to hang off system as it starts putting experimental references where they don't need to be. Easier to group under a single config.